### PR TITLE
User resets forgotten password

### PR DESCRIPTION
**Pull request by ghost**
### What we learned

- About mailer settings

### CODE REVIEW

File: features/step_definitions/assertion_steps.rb [hunk 1]
```diff
@@ -5,3 +5,7 @@
 Then("I should be redirected to index page") do
   expect(current_path).to eq root_path
 end
+
+Then("I am on password reset page") do
+  expect(current_path).to eq new_user_password_path
```
Comment (tochman) [inline features/step_definitions/assertion_steps.rb]: Wouldn’t it be better with “I should be...”?

File: features/user_logs_in_to_account.feature [hunk 2]
```diff
@@ -5,8 +5,8 @@ Feature: User logs in to account
 
 Background:
   Given We have the following user:
-    | name    | email           | password    |
-    | Maran   | maran@test.com  | 12345678    |
+    | email           | password    |
```
Comment (tochman) [inline features/user_logs_in_to_account.feature]: Thank you for correcting this. We missed it in code reviews. 👍

**Review by tochman — APPROVED** [review]
All good now.

**Review by GeorgeJLewis — APPROVED** [review]
LGTM

**Review by tochman — APPROVED** [review]
Looks good

**Pull request merged**